doc/dev/developer_guide/merging.rst

```diff
@@ -102,7 +102,7 @@ After Sage Weil announces that it is time for phase 2 to happen.
 Branch merges
```

Comment:
^ is the remark re Sage still valid?

Reply:
I was wondering this, too. I'm asking Josh and Neha in the morning.

Reply:
I might suspect s/Sage Weil/the Ceph Leadership Team/